### PR TITLE
refactor(scrapers): move scrapers under orgs

### LINKED ISSUE #13198: Make settings/scrapers org centric
Move `src/organizations/containers/OrgScrapersIndex.tsx` to `src/scrapers/containers/ScrapersIndex.tsx`
Use GetResources for loading data in to redux state instead of `GetOrgResources`
move all downstream components from OrgScrapersIndex to `scrapers/components`
Make sure all scraper fetching and all data fetching in this set of components is in org context make issues if client or API unavailable

### PR DESCRIPTION
**Pull request by AlirieGray**
Closes #13198
Closes https://github.com/influxdata/applications-team-issues/issues/299
Closes https://github.com/influxdata/applications-team-issues/issues/296
Closes https://github.com/influxdata/applications-team-issues/issues/295
Closes https://github.com/influxdata/applications-team-issues/issues/294

This PR moves the `ScrapersIndex` and sub-components to live under the /orgs route and also moves the overlays associated with Scrapers to live under their own routes.

  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

File: ui/src/buckets/components/BucketList.tsx [hunk 1]
```diff
@@ -131,38 +105,47 @@ class BucketList extends PureComponent<Props & WithRouterProps, State> {
...
+    onSetDataLoadersType(dataLoaderType)
+    router.push(this.overlayRoute(dataLoaderType, bucket.id))
   }
 
   private get isBucketOverlayVisible(): boolean {
     const {bucketID, bucketOverlayState} = this.state
     return !!bucketID && bucketOverlayState === OverlayState.Open
   }
 
+  private overlayRoute(
```
Comment (alexpaxton) [inline ui/src/buckets/components/BucketList.tsx]: This seems a bit like a router inside a router

File: ui/src/telegrafs/components/Collectors.tsx [hunk 2]
```diff
@@ -2,21 +2,18 @@
...
 
 // Components
 import {Input, Button, EmptyState, Grid} from '@influxdata/clockface'
 import {Tabs} from 'src/clockface'
 import CollectorList from 'src/telegrafs/components/CollectorList'
 import TelegrafExplainer from 'src/telegrafs/components/TelegrafExplainer'
-import TelegrafInstructionsOverlay from 'src/telegrafs/components/TelegrafInstructionsOverlay'
 import TelegrafConfigOverlay from 'src/telegrafs/components/TelegrafConfigOverlay'
-import CollectorsWizard from 'src/dataLoaders/components/collectorsWizard/CollectorsWizard'
+// import CollectorsWizard from 'src/dataLoaders/components/collectorsWizard/CollectorsWizard'
```
Comment (alexpaxton) [inline ui/src/telegrafs/components/Collectors.tsx]: Should this get removed?

File: ui/src/telegrafs/components/Collectors.tsx [hunk 3]
```diff
@@ -139,14 +136,7 @@ export class Collectors extends PureComponent<Props, State> {
...
         </Grid>
-        {this.collectorsWizard}
-        <GetResources resource={ResourceTypes.Authorizations}>
-          <TelegrafInstructionsOverlay
-            visible={this.isInstructionsVisible}
-            collector={this.selectedCollector}
-            onDismiss={this.handleCloseInstructions}
-          />
-        </GetResources>
+        {/* {this.collectorsWizard} */}
```
Comment (alexpaxton) [inline ui/src/telegrafs/components/Collectors.tsx]: Should this be removed?

Comment (AlirieGray) [reply ui/src/telegrafs/components/Collectors.tsx]: oops yes

File: ui/src/telegrafs/components/Collectors.tsx (hunk 2 above)
Comment (AlirieGray) [reply ui/src/telegrafs/components/Collectors.tsx]: yes, good catch

File: ui/src/buckets/components/BucketList.tsx (hunk 1 above)
Comment (AlirieGray) [reply ui/src/buckets/components/BucketList.tsx]: yes, that makes sense. I'll change that

File: ui/cypress/e2e/dashboardsIndex.test.ts [hunk 4]
```diff
@@ -51,24 +51,26 @@ describe('Dashboards', () => {
     cy.getByTestID('dashboard-card').should('have.length', 1)
   })
 
-  it('can create a dashboard from a Template', () => {
-    cy.getByTestID('dashboard-card').should('have.length', 0)
-    cy.get<Organization>('@org').then(({id}) => {
-      cy.createDashboardTemplate(id)
-    })
+  for (let i = 0; i < 99; i++) {
```
Comment (ebb-tide) [inline ui/cypress/e2e/dashboardsIndex.test.ts]: why is this in a for loop?

File: ui/src/scrapers/containers/ScrapersIndex.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+      </>
+    )
+  }
+}
+
+const mstp = (state: AppState, props: Props) => {
+  const {
+    orgs: {items},
+  } = state
+  const org = items.find(o => o.id === props.params.orgID)
```
Comment (ebb-tide) [inline ui/src/scrapers/containers/ScrapersIndex.tsx]: you can get org directly from redux state here. orgs:{org}

File: ui/src/dashboards/components/DashboardImportFromTemplateOverlay.tsx [hunk 6]
```diff
@@ -193,7 +193,7 @@ class DashboardImportFromTemplateOverlay extends PureComponent<
   private selectTemplate = (
     selectedTemplateSummary: TemplateSummary
   ) => async (): Promise<void> => {
-    this.setState({selectedTemplateSummary})
+    await this.setState({selectedTemplateSummary})
```
Comment (ebb-tide) [inline ui/src/dashboards/components/DashboardImportFromTemplateOverlay.tsx]: I don't understand why we are awaiting a state set?

**Pull request merged**